src/EgoErpArchiver/ViewModel/MainViewModel.cs

```diff
@@ -18,77 +18,62 @@ namespace EgoErpArchiver.ViewModel
     public class MainViewModel : ViewModelBase
     {
         #region Data
-        string windowTitle;
-        string filePath;
-        ErpFile file;
-        readonly ResourcesWorkspaceViewModel resourcesWorkspace;
-        readonly PackagesWorkspaceViewModel packagesWorkspace;
-        readonly TexturesWorkspaceViewModel texturesWorkspace;
-        readonly XmlFilesWorkspaceViewModel xmlFilesWorkspace;
+        private string windowTitle;
+        private string filepath;
```

Comment:
Nit: Rename to `_filePath` to follow editorconfig guidelines. I'm aware a lot of the existing code doesn't follow the guidelines.